services/pkg/man-cgi/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+for d in /man-cgi/pages/*; do
+    ln -s "$d" /var/lib/man-cgi/
+done
```

Comment:
this is to get around issues with mounting directly on `/var/lib/man-cgi/` overwriting the contents of the dir. should it go here or in the dockerfile?

Reply:
I'm not entirely sure what problem you're trying to get around?  Can you clarify what you're trying to resolve with this?